devtools/conda-envs/asapdiscovery.yaml

```diff
@@ -83,3 +89,5 @@ dependencies:
     # viz
     - pygifsicle
     - sigfig
+    - git+https://github.com/openforcefield/alchemiscale.git@main
+    - git+https://github.com/OpenFreeEnergy/cinnabar.git@main
```

Comment:
Why is it that we need the git cinnabar and not the package at conda-forge? Do we need to push for a new release for it? :)

Reply:
Yeah @ijpulidos this uses bleeding edge cinnabar code that isn't in the most recent conda package, so yes we should push for a release IMO if its not too much trouble.

Reply:
Same holds for alchemiscale, actually

Reply:
As far as I could tell there is not an alchemiscale package on conda I might have missed that though.

Reply:
https://anaconda.org/openforcefield/alchemiscale-client

Reply:
We do distribute on PyPI!

https://pypi.org/project/alchemiscale/

Sorry, forgot to push the 0.1.4 release; now it's there too!